eel/__init__.py

```diff
@@ -138,12 +138,18 @@ def run_lambda():
             HOST = '0.0.0.0'
         else:
             HOST = _start_args['host']
+
+        app = _start_args.get('app', btl.default_app)  # type: btl.Bottle
```

Comment:
Scince this field is already present, because it got set in line 43.
This line has to be `app = _start_args['app']` and line 43 `'app': btl.default_app(), `
the app will crash if run without an set app, because `'NoneType' object has no attribute 'route`.

Reply:
Gppd catch 👌